### ISSUE TITLE
[BUG]: tests fail in jest and jsdom -`fetch is not defined`

### ISSUE THREAD

**Issue by tay1orjones** [IBMa/equal-access#2283]
### Project

accessibility-checker for Node

### Browser

Chrome

### Operating system

MacOS

### Description

We updated to `accessibility-checker@4.0.4` and our jest tests fail.

```
    Expected test not to call console.log()

    ReferenceError: fetch is not defined

      47 |   const unexpectedConsoleCallStacks = [];
      48 |   const patchedConsoleMethod = function (format, ...args) {
    > 49 |     const stack = new Error().stack;
         |                   ^
      50 |     unexpectedConsoleCallStacks.push([
      51 |       stack.substr(stack.indexOf('\n') + 1),
      52 |       util.format(format, ...args),

      at node_modules/src-ts/lib/common/api-ext/Fetch.ts:18:18
      at node_modules/accessibility-checker/cjs/lib/common/api-ext/Fetch.js:23:71
      at Object.<anonymous>.__awaiter (node_modules/accessibility-checker/cjs/lib/common/api-ext/Fetch.js:19:12)
      at fetch_get (node_modules/accessibility-checker/cjs/lib/common/api-ext/Fetch.js:30:12)
      at node_modules/src-ts/lib/common/config/ACConfigManager.ts:163:43
      at node_modules/accessibility-checker/cjs/lib/common/config/ACConfigManager.js:41:71
      at Object.<anonymous>.__awaiter (node_modules/accessibility-checker/cjs/lib/common/config/ACConfigManager.js:37:12)
      at processACConfig (node_modules/accessibility-checker/cjs/lib/common/config/ACConfigManager.js:184:12)
      at node_modules/src-ts/lib/common/config/ACConfigManager.ts:434:11
      at fulfilled (node_modules/accessibility-checker/cjs/lib/common/config/ACConfigManager.js:38:58)
      at console.patchedConsoleMethod [as log] (config/jest-config-carbon/setup/setupAfterEnv.js:49:19)
      at node_modules/src-ts/lib/common/config/ACConfigManager.ts:165:17
          at Generator.throw (<anonymous>)
      at rejected (node_modules/accessibility-checker/cjs/lib/common/config/ACConfigManager.js:39:65)
      at formatConsoleCallStack (config/jest-config-carbon/setup/setupAfterEnv.js:91:9)
      at Object.formatConsoleCallStack (config/jest-config-carbon/setup/setupAfterEnv.js:73:7)
```

`accessibility-checker` [removed axios](https://github.com/IBMa/equal-access/pull/2253/files#diff-848072658be27a5edc8a17b857869de7dbe082b5f5bd438e4fa7fcf93be4599cL26) in favor of [using fetch directly](https://github.com/IBMa/equal-access/pull/2253/files#diff-05b826741d7c895927547e6f70e661776ce609de190e02a78ec23b879f9a863dR18) and unfortunately jsdom does not include fetch, https://github.com/jsdom/jsdom/issues/1724

We can polyfill it on our own, but since it's outside our scope of control I'd prefer a fix through `accessibility-checker` instead.

### Steps to reproduce

See https://github.com/carbon-design-system/carbon/pull/19251

Here's a link directly to the logs: https://github.com/carbon-design-system/carbon/actions/runs/14837801107/job/41653022384?pr=19251#step:8:6877

**Comment by philljenkins** [comment]
This should be fixed prior to next [release tgt 4.0.6](https://github.com/IBMa/equal-access/releases/) that will be deployed near the end of June.

**Comment by tombrunet** [comment]
Apparently a known issue with jsdom: https://github.com/jsdom/jsdom/issues/1724 - looking into workarounds.

**Comment by tombrunet** [comment]
Will be included in the next deployment
